runelite-client/src/main/java/net/runelite/client/plugins/tithefarm/TitheFarmInventoryOverlay.java

```diff
@@ -0,0 +1,60 @@
+package net.runelite.client.plugins.tithefarm;
+
+import net.runelite.api.Query;
+import net.runelite.api.queries.InventoryWidgetItemQuery;
+import net.runelite.api.widgets.WidgetItem;
+import net.runelite.client.ui.FontManager;
+import net.runelite.client.ui.overlay.Overlay;
+import net.runelite.client.ui.overlay.OverlayPosition;
+import net.runelite.client.ui.overlay.components.TextComponent;
+import net.runelite.client.util.QueryRunner;
+
+import javax.inject.Inject;
+import java.awt.Dimension;
+import java.awt.Graphics2D;
+import java.awt.Point;
+import java.awt.Rectangle;
+
+class TitheFarmInventoryOverlay extends Overlay
+{
+	private final QueryRunner queryRunner;
+	private final TitheFarmPluginConfig config;
+
+	@Inject
+	TitheFarmInventoryOverlay(QueryRunner queryRunner, TitheFarmPluginConfig config)
+	{
+		setPosition(OverlayPosition.DYNAMIC);
```

Comment:
After the update to change default overlay layer, this overlay layer should be set to ABOVE_WIDGETS